registry/remote/errcode/errors.go

```diff
@@ -16,13 +16,30 @@ limitations under the License.
 package errcode
 
 import (
+	"errors"
 	"fmt"
 	"net/http"
 	"net/url"
 	"strings"
 	"unicode"
 )
 
+const (
```

Comment:
should we attach the link of related distribution doc here?

Reply:
added!